webapp/sources/rudder/rudder-web/src/main/resources/logback.xml

```diff
@@ -57,6 +57,15 @@ along with Rudder.  If not, see <http://www.gnu.org/licenses/>.
    -->
 
 
+  <!--
+    Those properties defines the directory into which log using an appender will be written.
+    It can be overwritten when the application is launched with the
+    Java system property syntax: java -DWEBAPP_DIR="/some/other/directory/"
+  -->
+  <property name="APILOG_DIR" value="/var/log/rudder/api" />
```

Comment:
I think API could go under the webapp directory. It doesn't feel "business" at all (contrary to compliance), quite technical, and really linked to the webapp operations.

Reply:
I'm ok with you, it was done like this before, but yes it makes sense